src/python/WMCore/WMBS/MySQL/Create.py

```diff
@@ -65,6 +65,26 @@ def __init__(self, logger = None, dbi = None, params = None):
           """CREATE UNIQUE INDEX uniq_wmbs_file_run_lumi on
              wmbs_file_runlumi_map (fileid, run, lumi)"""
 
+        #
+        # Insert LogCollect site to account for LogCollect output
+        #
+        self.inserts["99_location_logcollect_1"] = \
```

Comment:
As-is right now the JobAccountant writes one LogCollect output file with LFN="srm castor pfn" location at T2_CH_CERN, which is complete bogus on various levels. The LFN is wrong, it's a PFN and it's a PFN that doesn't actually represent a file on T2_CH_CERN, but on T0_CH_CERN_MSS.